src/components/Events/CreateEvent.vue

```diff
@@ -0,0 +1,144 @@
+                <input v-validate="'required'"
+                v-model="date" name="date" type="date" size="50">
+                <input v-validate="'required|max:80'" v-model="location"
+                name="location" type="text" size="60" placeholder="Location">
+            </div>
+            <div class="form-errors">
+                <span v-show="errors.has('date')">{{ errors.first('date') }}</span>
+                <span v-show="errors.has('location')">{{ errors.first('location') }}</span>
+            </div>
+            <div class="form-element">
+                <input v-validate="'required'" v-model="startTime"
+                name="start time" type="time" step="300" size="30">
+                <p>to</p>
+                <input v-validate="'required'" v-model="endTime"
+                name="end time" type="time" step="300" size="30">
+            </div>
+            <div class="form-errors">
+                <span v-show="errors.has('start time')">{{ errors.first('start time') }}</span>
+                <span v-show="errors.has('end time')">{{ errors.first('end time') }}</span>
+            </div>
+            <div class="form-element">
+                <textarea v-validate="'required'" v-model="description" rows="10" cols="100"
+                name="description" placeholder="Description"></textarea>
+            </div>
+            <div class="form-errors">
+                <span v-show="errors.has('description')">{{ errors.first('description') }}</span>
+            </div>
+            <div class="form-element">
+                <textarea v-validate="'required'" v-model="whatToBring" rows="7" cols="100"
+                name="what to bring" placeholder="What to Bring"></textarea>
+            </div>
+            <div class="form-errors">
+                <span v-show="errors.has('what to bring')">
+                    {{ errors.first('what to bring') }}
+                </span>
+            </div>
+        </div>
+        <div class="buttons">
+            <input type="submit" class="button" value="Create">
+        </div>
+    </div>
+    </form>
+</template>
+
+<script>
+import Vue from 'vue';
+import VeeValidate from 'vee-validate';
+
+Vue.use(VeeValidate);
+export default {
+  name: 'NewEvent',
+  data() {
+    return {
+      event: {
+        name: '',
+        date: '',
+        startTime: '',
+        endTime: '',
+        description: '',
+        whatToBring: '',
+      },
+    };
+  },
+  methods: {
+    onSubmit() {
+      this.$validator.validateAll().then((result) => {
+        if (result) {
+          // add http post request
+          // eslint-disable-next-line
+          alert('create event successful');
+          this.$router.push('/events');
+        }
+      });
+    },
+  },
+};
+</script>
+
+<style scoped>
+@import '../../../assets/color-constants.less';
+.container {
+    margin: 3rem;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+}
+
+.event-name {
+    margin: 12px;
```

Comment:
Try to use rem in all these places like you did with .container. It's okay to have decimal rem values.

Reply:
thanks for the note, changed these!